openpecha/core/metadata.py

```diff
@@ -115,3 +115,42 @@ class DiplomaticPechaMetadata(PechaMetadata):
     @validator("id", pre=True, always=True)
     def set_id(cls, v):
         return v or ids.get_diplomatic_id()
+
+class AlignmentMetadata(BaseModel):
+    id:str
+    title:str
+    type: str
+    pechas: List[str]
+    alignment_to_base:Dict[str,str]
+    source_metadata: Optional[Dict] = None
+
+class InstanceMetadata(BaseModel):
+    id: str
+    title: List[str]
+    colophon: str
+    authors: List[str]
+    bdrc_id: str
+    location_info: dict
```

Comment:
what does that mean?

Reply:
The above attributes of Instance metadata are extracted from the using this [api](https://purl.bdrc.io/query/graph/OP_info?R_RES=bdr:{id}&format=ttl) pasing the instance id to it.The code is over [here](https://github.com/OpenPecha/Work-File-Creation/blob/main/parse_graph.py#L122).

Reply:
ok, but that's not really how it should work: the design of the metadata in OpenPecha shouldn't be driven by some query I made quickly a few months ago, there should be some proper design effort to create good OpenPecha metadata, document it and then build a query to get this data from BDRC